src/util.c

```diff
@@ -340,3 +340,65 @@ g_debug_tpma_cc (TPMA_CC tpma_cc)
     g_debug ("  V:            %s", prop_str (tpma_cc & TPMA_CC_V));
     g_debug ("  Res:          0x%" PRIx8, (tpma_cc & TPMA_CC_RES_MASK) >> TPMA_CC_RES_SHIFT);
 }
+/*
+ * Parse the provided string containing a key / value pair separated by the
+ * '=' character.
+ */
+gboolean
+parse_key_value (char *key_value_str,
+                 key_value_t *key_value)
+{
+    const char *delim = "=";
+    char *tok, *state;
+
+    g_debug ("%s: with key_value_str: \"%s\" and key_value_t: 0x%" PRIxPTR, __func__, key_value_str, (uintptr_t)key_value);
+    tok = strtok_r (key_value_str, delim, &state);
```

Comment:
Using strtok_r or strtok is dangerous as it destroys the input string (it replaces the delimiters with the NUL character).
This is OK as long as the caller expects it.  So I would add a note that the input string is modified around line 345.

Reply:
I see your point. The input parameter isn't `const` so the function prototype is clear on this point but lots of people assume otherwise. Adding a comment is easy enough.

Reply:
Thanks. Unfortunately, many C programmers are forget to add const, since it is a "new" feature (~20 years old) or are not always careful.  So the absence of const isn't always that useful.